gogs/inventory/hosts

```diff
@@ -0,0 +1,2 @@
+[seed-hosts]
+localhost
```

Comment:
Please change to `localhost ansible_connection=local`, otherwise the applier will fail with SSH errors.

Reply:
Done